### DISCUSSION TITLE
Are there any plans to make a new minor release to support Node 14?

### DISCUSSION THREAD

**Discussion by sryze** [gulpjs/gulp#2649] [Help] [answered]
Hi,

Since the watch function is not working well with Node 14.x and later, and Gulp 5/6 has been in the works for a long time now, can we maybe have a minor/patch release of Gulp 4.x with Node 14.x support? Or it requires a major rewrite?

Thanks.

**Comment by phated** [comment, marked as answer]
There are no plans for a minor release. There are too many breaking changes in the dependency chain that would not make this a semver-minor release. The team is working towards the 5.0 release which will be node 10+ and support node 14 & 16, etc.

**Reply by rogemus** [reply]
Hi @phated ,

Do you have some date/plan when you want to release v5?

**Comment by 0liver** [comment]
As a note to myself and other people coming here - the progress towards Gulp 5 can be monitored here: https://github.com/orgs/gulpjs/projects/4
